looprpc/client.proto

```diff
@@ -822,6 +822,15 @@ message LiquidityParameters {
     */
     uint64 autoloop_budget_sat = 11;
 
+    /*
+    Deprecated, use autoloop_budget_refresh_period_sec. The start time for
+    autoloop budget, expressed as a unix timestamp in seconds. If this value is
+    0, the budget will be applied for all automatically dispatched swaps. Swaps
+    that were completed before this date will not be included in budget
+    calculations.
+    */
+    uint64 autoloop_budget_start_sec = 12 [deprecated = true];
```

Comment:
I think ideally if we bring this back as `deprecated` we also want the functionality to work as then we cause old clients to silently just fail.

Reply:
I think this piece of code covers that case pretty well https://github.com/lightninglabs/loop/pull/559/commits/7dba48877b8348288a354ebca150b794a16c28fb#diff-cbbb732798eb63122c496cea51308ad255cb84338e14d1a0de26041353f69085R409-R420.

If somebody runs `loopd` with `autoloop` enabled then the above code will silently convert the old-style budget parameter to the new parameters being used.